src/api/server/ServerApi.ts

```diff
@@ -589,15 +590,14 @@ export default class ServerApi {
 
       const recipes = paths
         .map(id => {
-          let Recipe;
           try {
             // eslint-disable-next-line import/no-dynamic-require
-            Recipe = require(id)(RecipeModel);
+            const Recipe = require(id)(RecipeModel);
```

Comment:
should the variable be named `recipe` instead of `Recipe`?

Reply:
Generally I agree, but two lines below it is then used as `new Recipe` with the model, so should be correct.